### ISSUE TITLE
DPI Error: DPI-1037: column at array position fetched with error 1406

### ISSUE THREAD

**Issue by madmaxio** [kubo/rust-oracle#3]
Getting this while doing select, I beleive it is :

```
ORA-01406: fetched column value was truncated
Cause: In a host language program, a FETCH operation was forced to truncate a character string. The program buffer area for this column was not large enough to contain the entire string. The cursor return code from the fetch was +3.
Action: Increase the column buffer area to hold the largest column value or perform other appropriate processing. 
```

Any ideas how to fix?

**Comment by kubo** [comment]
@madmaxio I have not seen such error. Could you post the following information?
* Sample code to reproduce the issue
* The table definition. Run `desc TABLE_NAME` on sqlplus.
* The database NLS character set. Run `select value from nls_database_parameters where parameter='NLS_CHARACTERSET';` on sqlplus.

**Comment by kubo** [comment]
I reproduced it. You have no need to post the information I requested.

**Comment by kubo** [comment]
I think that this issue was fixed by 2e00b3a. Could you try it using the following `Cargo.toml`?
```
[dependencies]
oracle = { git = "https://github.com/kubo/rust-oracle.git" }
```

**Comment by madmaxio** [comment]
Worked! Thx.
